hpx/runtime/threads/run_as_hpx_thread.hpp

```diff
@@ -68,12 +71,12 @@ namespace hpx { namespace threads
                         std::lock_guard<hpx::lcos::local::spinlock> lk(mtx);
                         stopping = true;
                     }
```

Comment:
When this scope ends the lock is released and the spawning thread is free to continue past `cond->wait` (line 79 below). That leaves some time for `run_as_hpx_thread` to return before `cond->notify_all` gets called.

Reply:
Ok, thanks!